src/lib.rs

```diff
@@ -56,8 +57,6 @@ use std::fmt;
 use std::error;
 use libc::PATH_MAX;
 
-pub type Result<T> = result::Result<T, Error>;
```

Comment:
Based on what you said, and when I'm more awake, I'm pretty sure this will be fine. I'd been meaning to do a `from_ffi` sweep soon, so...